addon/ng2/blueprints/ng2/files/package.json

```diff
@@ -21,21 +21,21 @@
     "@angular/platform-browser": "2.0.0-rc.4",
     "@angular/platform-browser-dynamic": "2.0.0-rc.4",
     "@angular/router": "3.0.0-beta.2",
-    "es6-shim": "0.35.1",
-    "reflect-metadata": "0.1.3",
+    "ts-helpers": "^1.1.1",
     "rxjs": "5.0.0-beta.6",
     "systemjs": "0.19.26",
-    "zone.js": "0.6.12"
+    "zone.js": "0.6.12",
+    "core-js": "^2.4.0"
```

Comment:
You know how I know you didn't use `npm install --save` ?  Order everything alphabetically.